### DISCUSSION TITLE
Securitonion 2.4 Beta3 - couldn't install under Virtualbox

### DISCUSSION THREAD

**Discussion by ejgh-oe** [Security-Onion-Solutions/securityonion#10517] [2.4]
Hi,

Today I tried to get SO 2.4 Beta 3 running under Virtualbox but failed miserably :-(

The details:
Virtualbox 6.1.44 running under Ubuntu 20.04: 8GB RAM, 200GB diskspace, 2 virtual CPUs, network in bridged mode

The first part of the installation, i.e. copying the contents of the ISO to the VM went OK, finishing up with "Initial Install Complete. Press [Enter] to reboot!"

After rebooting however the VM doesn't come up throws error messages wrt /dev/mapper and after about 30sec ends up in "emergency" mode. No login prompt, nothing. When hitting Enter the exact same error message about "emergency mode" is displayed again plus a message that the root account is locked.

Rebooting via power-cycling the VM doesn't change anything - same behavior / error messages as before.

I'm attaching screenshots showing what's going on here.

Any ideas as to what could be wrong? Shouldn't the Beta3 ISO work under Virtualbox?

Thanks much in advance for any clue,
-ewald

![so-24-beta3-01](https://github.com/Security-Onion-Solutions/securityonion/assets/84927703/0187ef67-9289-4b56-a91c-b7aeccefc8cd)

![so-24-beta3-02](https://github.com/Security-Onion-Solutions/securityonion/assets/84927703/044570e0-9cf9-433e-bc95-6387f2b07c6f)

![so-24-beta3-03](https://github.com/Security-Onion-Solutions/securityonion/assets/84927703/a1d2e2a2-4675-4165-a1db-5afdfaca3cf9)

**Comment by dougburks** [comment]
Did you verify the ISO image?
https://github.com/Security-Onion-Solutions/securityonion/blob/2.4/main/VERIFY_ISO.md

What are the specs of your host machine?

**Reply by ejgh-oe** [reply]
Hi,
Yes the checksum of the ISO is correct:

```
$ sha256sum securityonion-2.4.2-20230531.iso
b69c1ae4c576bbbc37f4b87c2a8379903421e65b2c4f24c90fabb0ead6f0471b  securityonion-2.4.2-20230531.iso
$

```
Specs of my host machine:
RAM: 32GB
SSD: 800GB
CPU: Intel(R) Core(TM) i5-7500 CPU @ 3.40GHz

BTW, no problem running the latest SO 2.3 on that machine with the exact same virtual hardware configuration outlined above.

**Reply by dougburks** [reply]
I've duplicated the problem and created an issue to work on it:
https://github.com/Security-Onion-Solutions/securityonion/issues/10534

**Reply by dougburks** [reply]
This will be resolved in the next ISO image.

**Reply by ejgh-oe** [reply]
Thanks alot Doug - I'll be definitely checking it out once it's released ;-)

**Comment by petiepooo** [comment]
For the sake of completeness, if you are in this situation and cannot boot, here is what Rocky recommends to continue past the error:

> In the case of a system which has already been upgraded and is in recovery mode, you may simply move the /etc/lvm/devices/system.devices file out of the way from recovery mode and reboot the system. Once the system has rebooted, run 'vgimportdevices --all' to regenerate the file in the new format.

**Comment by ejgh-oe** [comment]
Ah, good to know. I'll give it a shot, even though my installation was only for testing.

**Comment by ejgh-oe** [comment]
Works - thanks for the update
